Content.Server/Lathe/LatheSystem.cs

```diff
@@ -182,7 +182,9 @@ public bool TryStartProducing(EntityUid uid, LatheComponent? component = null)
         {
             if (!Resolve(uid, ref component))
                 return false;
-            if (component.CurrentRecipe != null || component.Queue.Count <= 0 || !this.IsPowered(uid, EntityManager))
+            if (component.CurrentRecipe != null || component.Queue.Count <= 0)
+                return false;
+            if (HasComp<ApcPowerReceiverComponent>(uid) && !this.IsPowered(uid, EntityManager))
```

Comment:
Is Powered should just return true if the entity does not have the component

Reply:
yes this pr was not needed